src/components/resource/edit-custom/custom-resource-edit.js

```diff
@@ -59,7 +60,6 @@ class CustomResourceEdit extends Component {
   render() {
     let {
       model,
-      initialValues,
```

Comment:
I think this needs to come back. Probably just got lost in a rebase.

Reply:
Whoops, yup. I for sure didn't do that intentionally haha. Good catch.